pkg/pss/utils/mapping.go

```diff
@@ -0,0 +1,626 @@
+package utils
+
+var PSS_baseline_control_names = []string{
```

Comment:
Global variables should be avoided, but we can rework this later.

Reply:
This is used in `/api` for validation, I was thinking to add this under `/api` but wanted to define all PSS paras in a single file.